src/v2/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -786,7 +793,6 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
                 <Button
                   onClick={this.onContinueButtonPressed}
                   loading={isCommittingMutation}
-                  disabled={isContinueButtonDisabled}
                   variant="primaryBlack"
```

Comment:
TO DO: we need to ticket better error handling and disabled state, in this PR we are removing the disabled state for the `continue` button due to the new auto quote selection logic. We need to disable in the correct spot where there are actually no quotes returned from arta OR the request to arta failed.

Reply:
To clarify - that would be a new functionality that is why we are not tying it to this bug. In current production version 'continue' button is NOT disabled where arta returns no quotes or errors.